### PR TITLE
Fix merge comment - CLI mode

### PR DESCRIPTION
**Pull request by AsafCX**
### Description

Needed to add merge id in request metadata in order to update an existing comment

### References

https://github.com/checkmarx-ltd/cx-flow/issues/655

### Checklist

- [ ] I have added documentation for new/changed functionality in this PR (if applicable).  *If documentation is a Wiki Update, please indicate desired changes within PR MD Comment*
- [X] All active GitHub checks for tests, formatting, and security are passing
- [X] The correct base branch is being used

### CODE REVIEW

**Comment by cxflowtestuser** [comment]
Scan submitted to Checkmarx

**Comment by cxflowtestuser** [comment]
Logo<br />**Checkmarx SAST - Scan Summary & Details**
### Cx-SAST Summary
Total of 1168 vulnerabilities<br />High&nbsp;**0 High**<br />Medium&nbsp;**3 Medium**<br />Low&nbsp;**1091 Low**<br />Info&nbsp;**74 Info**<br />
### Violation Summary
Medium&nbsp;**3** **Medium**<br />
[View more details on Checkmarx UI](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1054857&projectid=1404)
### Cx-SAST Details
|Lines|Severity|Category|File|Link|
---|---|---|---|---
[67](https://github.com/checkmarx-ltd/cx-flow/blob/pr-Asaf-Fix-Gitlab-CLI-comment/src/main/java/com/checkmarx/flow/controller/FlowController.java#L67) [70](https://github.com/checkmarx-ltd/cx-flow/blob/pr-Asaf-Fix-Gitlab-CLI-comment/src/main/java/com/checkmarx/flow/controller/FlowController.java#L70) |Medium|Input_Path_Not_Canonicalized|src/main/java/com/checkmarx/flow/controller/FlowController.java|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1054857&projectid=1404&pathid=7)
[60](https://github.com/checkmarx-ltd/cx-flow/blob/pr-Asaf-Fix-Gitlab-CLI-comment/src/main/java/com/checkmarx/flow/service/ExternalScriptService.java#L60) |Medium|Input_Path_Not_Canonicalized|src/main/java/com/checkmarx/flow/service/ExternalScriptService.java|[Checkmarx](https://sast.checkmarx.net/CxWebClient/ViewerMain.aspx?scanid=1054857&projectid=1404&pathid=9)

File: src/main/java/com/checkmarx/flow/controller/GitLabController.java [hunk 1]
```diff
@@ -40,6 +40,8 @@
     private static final String EVENT = "X-Gitlab-Event";
     private static final String PUSH = EVENT + "=Push Hook";
     private static final String MERGE = EVENT + "=Merge Request Hook";
+    private static final String MERGE_ID = "merge_id";
```
Comment (alex-ko-dev) [inline src/main/java/com/checkmarx/flow/controller/GitLabController.java:43]: This constant is defined 3 times in the project. It should be defined at a single place.

Comment (AsafCX) [reply src/main/java/com/checkmarx/flow/controller/GitLabController.java]: Ok, Using FlowConstants as a single point definition

**Review by alex-ko-dev — DISMISSED** [review]

**Review by AvivCx — APPROVED** [review]

**Pull request merged**